app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -24,23 +24,30 @@
 import java.io.IOException;
 import java.util.Random;
 
+import javax.inject.Inject;
+
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import dagger.Binds;
 import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.auth.SessionManager;
 import fr.free.nrw.commons.theme.NavigationBaseActivity;
 
+/**
+ *  activity for sharing feedback on uploaded activity
+ */
 public class AchievementsActivity extends NavigationBaseActivity {
 
     private static final double BADGE_IMAGE_WIDTH_RATIO = 0.4;
     private static final double BADGE_IMAGE_HEIGHT_RATIO = 0.3;
 
     @BindView(R.id.achievement_badge) ImageView imageView;
     @BindView(R.id.toolbar) android.support.v7.widget.Toolbar toolbar;
```

Comment:
Why does `android.support.v7.widget.Toolbar` have the full path? It can be imported.